redhat-operators/codeready-workspaces/codeready-workspaces.1.1.0.csv.yaml

```diff
@@ -0,0 +1,343 @@
+                "pvcStrategy":"common",
+                "pvcClaimSize":"1Gi",
+                "preCreateSubPaths": true
+             }
+          }
+        }
+      ]
+    certified: "true"
+    categories: "Developer Tools"
+    containerImage: quay.io/crw/operator-container:1.1-7
+    createdAt: 2019-03-06
+    description: A collaborative kubernetes-native development solution that delivers OpenShift workspaces and in-browser IDE for rapid cloud application development.
+    support: Red Hat, Inc.
+spec:
+  install:
+    strategy: deployment
+    spec:
+      clusterPermissions:
+        - serviceAccountName: codeready-operator
+          rules:
+            - apiGroups:
+                - extensions/v1beta1
+              resources:
+                - ingresses
+              verbs:
+                - "*"
+            - apiGroups:
+                - route.openshift.io
+              resources:
+                - routes
+              verbs:
+                - "*"
+            - apiGroups:
+                - rbac.authorization.k8s.io
+              resources:
+                - roles
+                - rolebindings
+                - clusterroles
+                - clusterrolebindings
+              verbs:
+                - "*"
+            - apiGroups:
+                - ""
+              resources:
+                - pods
+                - services
+                - serviceaccounts
+                - endpoints
+                - persistentvolumeclaims
+                - events
+                - configmaps
+                - secrets
+                - pods/exec
+                - pods/log
+              verbs:
+                - '*'
+            - apiGroups:
+                - ""
+              resources:
+                - namespaces
+              verbs:
+                - get
+            - apiGroups:
+                - apps
+              resources:
+                - deployments
+              verbs:
+                - '*'
+            - apiGroups:
+                - monitoring.coreos.com
+              resources:
+                - servicemonitors
+              verbs:
+                - get
+                - create
+            - apiGroups:
+                - org.eclipse.che
+              resources:
+                - '*'
+              verbs:
+                - '*'
+      deployments:
+        - name: codeready-operator
+          spec:
+            replicas: 1
+            selector:
+              matchLabels:
+                app: codeready-operator
+            template:
+              metadata:
+                labels:
+                  app: codeready-operator
+              spec:
+                containers:
+                  - name: codeready-operator
+                    image: quay.io/crw/operator-container:1.1-7
+                    env:
+                      - name: WATCH_NAMESPACE
+                        valueFrom:
+                          fieldRef:
+                            fieldPath: metadata.namespace
```

Comment:
This should be set to `metadata.annotations['olm.targetNamespaces']`

Reply:
@SamiSousa thanks, fixed.